### DISCUSSION TITLE
Changing Output Directory?

### DISCUSSION THREAD

**Discussion by RexGhidorah** [anidl/multi-downloader-nx#764] [Q&A]
Hi!
Is there a way to custom the output direction? Like, instead to /video it goes to my plex folder hahaha
And... Automatically making a folder with the series name.

**Comment by Tama47** [comment]
Change the download path in `./config/dir-path.yml`

> And... Automatically making a folder with the series name.

At that point, you might just want to use Sonarr...

I just change the download path to my download folder and drag the episode there manually...
